kubernetes-services/work-scheduler/package.json

```diff
@@ -11,8 +11,8 @@
     "test-fast": "TS_NODE_TRANSPILE_ONLY=true mocha",
     "lint": "eslint --ext .ts .",
     "coverage": "nyc mocha",
-    "build": "tsc && docker build -f Dockerfile.mac --tag harmonyservices/work-scheduler:${VERSION:-latest} .",
-    "build-m1": "tsc && docker build -f Dockerfile.mac --platform linux/amd64 --tag harmonyservices/work-scheduler:${VERSION:-latest} .",
+    "build": "tsc && cd ../.. && docker build -f kubernetes-services/work-scheduler/Dockerfile.mac --tag harmonyservices/work-scheduler:${VERSION:-latest} .",
```

Comment:
Won't this change your directory after running? If so can we put the`cd` in a subshell - ie `(cd ...)`?

Reply:
`npm run` is already in a subshell, right?